test/corpus/insert.txt

```diff
@@ -526,3 +526,228 @@ FROM
                     (identifier))
                   (literal)))))
           (identifier))))))
+
```

Comment:
we've only got two tests for `merge` so far but if we expand them we'll want to have a dedicated merge.txt so why not start now?